SRTNet.h

```diff
@@ -114,6 +115,9 @@ class SRTNet {
      * @param overhead % extra of the BW that will be allowed for re-transmission packets
      * @param ctx the context used in the receivedData and receivedDataNoCopy callback
      * @param mtu sets the MTU
+     * @param failOnConnectionError if set to true and the initial connection attempt to the server fails this function
+     * will return false, if set to false the client will start an internal thread that will keep on trying to connect
+     * to the server until stop() is called.
```

Comment:
Also add to this documentation that if failOnConnectionError is set to true, the internal thread will keep trying to reconnect.

Reply:
Hm, ah, you mean in case the initial connection is ok, but the connections is broken later.. will clarify that :+1: